code/game/machinery/kitchen/microwave.dm

```diff
@@ -341,6 +348,31 @@
 
 	var/datum/recipe/recipe = select_recipe(available_recipes,src)
 	var/obj/cooked
+
+	for(var/obj/O in contents)
+		if(istype(O,/obj/item/weapon/kitchen/utensil) && !(O.melt_temperature == MELTPOINT_PLASTIC))
+			if (!wzhzhzh(4))
+				abort()
+				return
+			wzhzhzh(4)
+			stop()
```

Comment:
all this will do is play the dingdong, everything else is also handled in broke(). Intentional?

Reply:
fixed